vault/templates.topic.md

```diff
@@ -24,7 +24,7 @@ See [[Top Level Feature|dendron://dendron.dendron-site/dendron.contribute.docume
 
 ## Use Cases
 {{! 
-See [[Use Cases|dendron://dendron.handbook/leaflet.journal.2022.05.10.standard-documentation.topics#use-cases]]
```

Comment:
this link is actually correct - unfortunately doesn't resolve when you just pull in the dendron site vault by itself. we need to add instructions that working on docs also requires pulling in https://github.com/dendronhq/handbook